settings.py

```diff
@@ -39,6 +39,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'eventtracking.django',
```

Comment:
Do you actually still need this after putting in the mocks?

I'm not sure what this settings file is used for, but I suspect you could get away without putting in any of the event tracking settings.